<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen_In</class>
 <widget class="QMainWindow" name="Screen_In">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 198, 198);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_Out">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);
font: 20pt &quot;Arial&quot;;
font: bold;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>          LogOut</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ornament_Hello">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);
font: 20pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Bem vindo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/login.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Ornament_Functional">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);
font: 20pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Delete">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 0, 0);
font: 20pt &quot;Arial&quot;;
font: bold;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Deletar Conta</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
